src/main.rs

```diff
@@ -72,6 +72,10 @@ struct Opts {
     /// Path to graphql file containing Function input query; if omitted, defaults will be used to calculate limits.
     #[clap(short = 'q', long)]
     query_path: Option<PathBuf>,
+
+    /// Return a non-zero exit code when a module error occurs during the function run.
+    #[clap(short = 'n', long)]
+    non_zero_exit_code_for_module_errors: bool,
```

Comment:
This seems like a pretty long flag name to me. Unfortunately I don't have a constructive suggestion for something more succinct.